public/mobile-app/src/lib/state/User.svelte.ts

```diff
@@ -97,6 +98,8 @@ export class User {
       preferred_username: this._pivot.preferred_username,
       email: parsedIdentity?.email || this._pivot.email,
       address: parsedIdentity?.address,
+      scheduledNotificationsCreatedKeys:
+        parsedIdentity?.scheduledNotificationsCreatedKeys,
```

Comment:
This is weird, because it could be undefined, right? But the type line 40 doesn't say it can be undefined? No errors from the type checker? Am I missing something?

Reply:
Good point, I'll change it to:
```
      scheduledNotificationsCreatedKeys:
        parsedIdentity?.scheduledNotificationsCreatedKeys || [],
```
However, there were no typing errors or errors in the unit tests or in real life, but it's better th